python/modules/IcePy/Init.cpp

```diff
@@ -171,7 +165,7 @@ static struct PyModuleDef iceModule = {
     nullptr,
     nullptr,
     nullptr,
-    nullptr};
+    IcePy_cleanup};
```

Comment:
We were calling the cleanup function from Ice/__init__.py. I move this to the module-free function.